### PR TITLE
acceptance tests

### PR DESCRIPTION
**Pull request by bboozzoo**
@mendersoftware/rndity @GregorioDiStefano @maciejmrowiec

### CODE REVIEW

**Comment by bboozzoo** [comment]
Anyone else wants to comment on this? If not I'm merging

**Comment by GregorioDiStefano** [comment]
@bboozzoo please give me sometime to look into this, I'll take a look Monday.

**Comment by bboozzoo** [comment]
ok

File: tests/tests/test_token.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        rsp = requests.post(verify_url, data='',
+                            headers={'Authorization': 'bogus'})
+        assert rsp.status_code == 401
+
+        # or a correct token with data appended at the end
+        rsp = requests.post(verify_url, data='',
+                            headers={'Authorization': auth_hdr + "==foo"})
+        assert rsp.status_code == 401
+
+        # bravado cannot handle DELETE requests either
```
Comment (GregorioDiStefano) [inline tests/tests/test_token.py]: Yup, I can see examples in their test where they've used DELETE created from yaml: https://github.com/Yelp/bravado/blob/39814262547f7c331ff7aa9644e286adfa841969/tests/petstore/pet/deletePet_test.py

**Comment by GregorioDiStefano** [comment]
can you add some test verifying that the sequence number is working correctly?

**Comment by bboozzoo** [comment]
Misclicked and merged a this PR instead of another one. I'll add sequence number checks in separate pull request.

**Pull request merged**